spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/sql.adoc

```diff
@@ -322,6 +322,21 @@ You can customize the console's path by using the configprop:spring.h2.console.p
 
 
 
+[[features.sql.h2-web-console.spring-security]]
+==== Configuring Spring Security for H2 Console
+H2 Console uses frames and, as it's intended for development only, does not implement CSRF protection measures. If your application uses Spring Security, you need to configure it accordingly.
+
+For example, Spring Security will ignore the console if the following `WebSecurityCustomizer` is exposed:
```

Comment:
I think we need to be more explicit than "ignoring" and mention anybody could access it.